rest/komand_rest/util/util.py

```diff
@@ -22,9 +27,100 @@ def copy_dict(x):
         return d
 
     @staticmethod
-    def call_api(url, path, headers, ssl_verify, auth=None):
+    def body_object(response) -> object:
```

Comment:
```suggestion
    def body_object(response) -> dict:
```
`response.json()` yields a dictionary :)